src/rpc/core_rpc_server.cpp

```diff
@@ -2086,38 +2089,17 @@ namespace cryptonote
 
```

Comment:
I wondered about it, but since a lot of cases will be a wallet asking for 1200 (or so) blocks from the same height (the v6 chain fork), I thought it'd still help.